src/Pages/AppPage/AppGamePage/AppGamePage.js

```diff
@@ -24,6 +25,7 @@ const AppGamePage = function () {
         <SongSelector player={RIGHT_PLAYER}/>
       </div>
       <HelpModal/>
+      <Toast/>
```

Comment:
Maybe `<Toast />` could be included in the `index.js` instead, so that toasts are available on all pages?

Reply:
Agree this makes senese